draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -1114,21 +1118,24 @@ provides the security properties described above.
 
 ### IND-CCA analyses
 
-The QSF construction has two complementary IND-CCA analyses. Both were given
-in {{XWING}}. We summarize them but elide some details.
+The UG construction has two complementary IND-CCA analyses: one for when the SDH problem
+holds but the PQ KEM is broken, and one for the reverse. Both are technically novel but
+are substantially similar to the existing peer-reviewed analyses of the CG {{XWING}} and
+UK {{GHP18}} constructions. A forthcoming document by the editorial team will describe
+the analysis of UG in detail.
 
-One analysis (Theorem 1) {{XWING}} shows that if the KDF is modelled as a RO,
-IND-CCA holds if the PQ KEM is broken, as long as the SDH problem holds in
-the nominal group and the PQ KEM satisfies C2PRI. The other (Theorem 2)
-{{XWING}} shows that if the PQ-KEM is IND-CCA and the KDF is a PRF keyed on
-the PQ-KEM's shared secret, IND-CCA holds.
+The first IND-CCA analysis, based on SDH, is very similar to the corresponding
+analysis of CG given in {{XWING}}: it gives a straightforward reduction to the SDH
+hardness in the underlying group. Notably, since the PQ KEM key and ciphertext are hashed,
+the C2PRI security of the PQ KEM does not appear in the bound.
```

Comment:
```suggestion
The first IND-CCA analysis, based on SDH, is very similar to the corresponding
analysis of CG given in {{XWING}}: it gives a straightforward reduction to the
SDH hardness in the underlying group. Notably, since the PQ KEM key and ciphertext are
hashed, the C2PRI security of the PQ KEM does not appear in the bound.
```